dotnet/src/Connectors/Connectors.AI.OpenAI/ChatCompletionWithData/AzureOpenAIChatCompletionWithData.cs

```diff
@@ -343,10 +295,10 @@ private static OpenAIChatHistory InternalCreateNewChat(string? text = null, Open
         }
 
         // Try to parse the text as a chat history
-        if (XmlPromptParser.TryParse(text!, out var nodes) && ChatPromptParser.TryParse(nodes, out var chatHistory))
-        {
-            return new OpenAIChatHistory(chatHistory);
-        }
+        //if (XmlPromptParser.TryParse(text!, out var nodes) && ChatPromptParser.TryParse(nodes, out var chatHistory))
+        //{
+        //    return new OpenAIChatHistory(chatHistory);
+        //}
```

Comment:
commented out?

Reply:
Yep, I will remove this since the XmlPromptParser is now an internal implementation in the SK.Abstractions (not reacheable by connectors)